
public class TestResource {
	
	public void testFunction() {
		try {
			
		}
		catch (Exception e) {
			
		}
	}
}
